lib/screens/mobile/sneakers_grid.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_internship/data/sneakers_list.dart';
+import 'package:flutter_internship/widgets/components/product_card.dart';
+
+class SneakersGrid extends StatelessWidget {
+  const SneakersGrid({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    double screenWidth = MediaQuery.sizeOf(context).width;
+    int columnsCount = (screenWidth / 168).floor();
+
+    return SingleChildScrollView(
+      child: Container(
+        color: const Color(0xFFF8F8F8),
+        child: Padding(
+          padding: const EdgeInsets.all(20),
+          child: Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            children: [
+              const Text(
+                'Sneakers',
+                style: TextStyle(
+                    color: Color(0xFF000000),
+                    fontFamily: 'Inter',
+                    fontSize: 20,
+                    fontStyle: FontStyle.normal,
+                    fontWeight: FontWeight.w600,
+                    decoration: TextDecoration.none),
+              ),
```

Comment:
"Sneakers" should be in AppBar not just text in Column. 
ProductCard size is different on different width of application window.
There is black space when I'm on this view, see image, this happens when I click View all.
<img width="1317" alt="Screenshot 2024-01-30 at 09 26 19" src="https://github.com/damjan-Igrowth/flutter-internship/assets/119666343/e77bbdd6-e165-498e-b5c7-b8d5b661580d">